packages/grafana-ui/src/components/BarChart/bars.ts

```diff
@@ -235,50 +190,48 @@ export function getConfig(opts: BarsOptions) {
   };
 
   // handle hover interaction with quadtree probing
-  const setCursor = (u: uPlot) => {
-    let found: Rect | null = null;
-    let cx = u.cursor.left! * pxRatio;
-    let cy = u.cursor.top! * pxRatio;
-
-    qt.get(cx, cy, 1, 1, (o) => {
-      if (pointWithin(cx, cy, o.x, o.y, o.x + o.w, o.y + o.h)) {
-        found = o;
-      }
-    });
+  const interpolateBarChartTooltip: TooltipInterpolator = (
+    updateActiveSeriesIdx,
+    updateActiveDatapointIdx,
+    updateTooltipPosition
+  ) => {
+    return (u: uPlot) => {
+      let found: Rect | null = null;
+      let cx = u.cursor.left! * devicePixelRatio;
+      let cy = u.cursor.top! * devicePixelRatio;
+
+      qt.get(cx, cy, 1, 1, (o) => {
+        if (pointWithin(cx, cy, o.x, o.y, o.x + o.w, o.y + o.h)) {
+          found = o;
+        }
+      });
 
-    if (found) {
-      // prettier-ignore
-      if (found !== hovered) {
-        /* eslint-disable no-multi-spaces */
```

Comment:
same comment as the other one. this is not better, imo.

Reply:
yeah, u r right, it's me mindlessly removing ignores :)